### ISSUE TITLE
pychess default save location

### ISSUE THREAD

**Issue by axet** [pychess/pychess#2010]
Default save location for game history set to $HOME. This is very not common for linux apps.

Can default location be changed to default config directory (my optinion this is the best option) '$HOME/.config/pychess' or My Documents $(xdg-user-dir DOCUMENTS)?

pychess-1.0.3

**Comment by hartwork** [comment]
Hi! Git `master` is using `~/.config/pychess` for a number of things. Where did you see use of plain `~/`?

**Comment by axet** [comment]
I start fresh pychess-1.0.3 and it starts to pollute my home folder. As turns out in Edit / Preferencies / Save I found checked "Auto save finished game" set and pointing to $HOME.

This is not right way of working with user files in linux. Better to have this path changed to ~/.config/pychess by default.

EDIT: I changed it to the .config.pychess for for some reason path got reset and restored to the $HOME so I have to uncheck "Auto save finished games".

**Comment by axet** [comment]
Just test it with 'master' same. Fresh (after rm -rf .config/pychess) install result in "Auto save" pointing to the $HOME

**Comment by hartwork** [comment]
@axet I understand your message now. I see these files for me, you probably have the first two only:

```console
# ( cd && ls -1 pychess* )
pychess.pgn
pychess.sqlite
pychess_test.pgn
pychess_test.sqlite
```

For anyone else coming here, the place we are talking about is this:

![Screenshot_20230319_174846](https://user-images.githubusercontent.com/1577132/226191814-133e8564-b5c0-4109-bdf4-f9b8eabbc74c.png)

**Comment by gbtami** [comment]
I'm not really familiar with Linux file system standards, but you are right and it would be better to provide default to Documents (if it is the standard).

**Comment by axet** [comment]
No apps create files automatically in $HOME or Documents. Only when manually specified / saved. Default should go into ~/.config or ~/.cache (temporary files). Save or export game history should be offered into Documents.

**Comment by hartwork** [comment]
I disagree about some details here (..), but point taken that `${HOME}` is not an ideal choice.

**Comment by ydebilloez** [comment]
Please find patch that could be put in the source code. It does not fix the upgrade from a new installation but it fixes most of the issues in new installation. Still some fix required on preferences/save dialog initialisation.
[linuxdefaultfolderpatch.txt](https://github.com/user-attachments/files/17364155/linuxdefaultfolderpatch.txt)

**Comment by gbtami** [comment]
@ydebilloez may I ask you to create a pull request instead, please?
